src/Layouts/RightSideBar.vala

```diff
@@ -49,7 +49,7 @@ public class Akira.Layouts.RightSideBar : Gtk.Grid {
 	construct {
 		get_style_context ().add_class ("sidebar-r");
 		width_request = 220;
-		
```

Comment:
Again, revert this file please

Reply:
Hi @Philip-Scott, thanks for this extensive review, I'm really grateful for the time you put into this. 

I'm a total beginner with open source contributing, so that's why I make those mistakes. 

I just need some help with regards to this *revert* thing. What do I have to do, precisely? 
Do I have to use `git revert` or `git checkout` to the right branch? 

I was about to do something like that: 
```
git checkout 368ddf9 src/Layouts/RightSideBar.vala
```
with `368ddf9` being the commit `368ddf9 Fix Huston CI with Travis (#100)` which is the commit I  merged the first time you reviewed this PR. 

The problem is that with `git diff` I see no changes, not even the spacing you mentioned. 

Is this right? Do I have to change the commit hash, revert to another commit or use another command? 

Thanks for being so kind to me, I'm still new to all those things but I learn fast :)

Reply:
Hi @giacomoalbe, no problem at all! I'm always happy to help :)

When you do a checkout like that, the file you commit is already "staged", so it won't appear in `git diff` but the file will show up if you do `git status` under `Changes to be committed:`.

That said, you can run `git diff --staged` to get the changes you already added to the current commit, such as that one :)